patches/react-native-file-logger+0.4.1.patch

```diff
@@ -10,7 +36,7 @@ index 7ee491d..978ce4e 100644
  
  enum LogLevel {
      LOG_LEVEL_DEBUG,
-@@ -19,6 +20,17 @@ @interface FileLogger () <MFMailComposeViewControllerDelegate>
+@@ -19,6 +20,17 @@ static const DDLogLevel ddLogLevel = DDLogLevelDebug;
```

Comment:
I guess this is because I didn't run a pod install? No iOS changes are expected or wanted